src/main/webapp/cluster-view/cluster-view.less

```diff
@@ -29,18 +18,3 @@
   float: right;
   border: 1px solid #D8D8D8;
 }
-.card-link {
```

Comment:
@tristantarrant this was a request from Rome f2f (click anywhere on the card to activate link)